info.json

```diff
@@ -1,15 +1,60 @@
 {
     "name": "multiTenancy",
     "version": "1.0.0",
+    "type": "solutionpack",
+    "local": false,
     "label": "Multi Tenancy",
-    "description": "FortiSOAR Solution Pack for Multi Tenancy Use Cases",
+    "dependencies": [
+        {
+            "name": "fortiSOAREssentials",
+            "type": "solutionpack",
+            "label": "FortiSOAR Essentials"
+        },
+        {
+            "name": "socSimulator",
```

Comment:
Why MSSP is dependent on SoCSim?

Reply:
if we want to execute any use-case then for that we have to create the record for same and hence we required soc simulator SP

Reply:
MSSP/MTT is a framework SP and not a scenario pack. Please correct me if i am wrong here?

Reply:
For now, we will keep this dependency but lets review the SP content and figure out why we need SoC Sim here.